### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by rPraml**
Hello Rob,

a very big pull request that fixes a lot of (eclipse) warnings. There is one real code change in PlatformDdlWriter, all other changes should not affect existing code.

### CODE REVIEW

**Comment by rPraml** [comment]
Hello Rob, should I clean up/split this PR?

File: src/main/java/io/ebean/dbmigration/DbMigration.java [hunk 1]
```diff
@@ -431,20 +431,18 @@ protected void writeExtraPlatformDdl(String fullVersion, CurrentModel currentMod
 
     for (Pair pair : platforms) {
       DdlWrite platformBuffer = new DdlWrite(new MConfiguration(), currentModel.read());
-      PlatformDdlWriter platformWriter = createDdlWriter(pair);
+      PlatformDdlWriter platformWriter = createDdlWriter(pair.platform);
```
Comment (rbygrave) [inline src/main/java/io/ebean/dbmigration/DbMigration.java]: I'm not that happy when a PR contains multiple changes and is so massive.  This one contains a whole bunch of @Override ... but then also includes a design change for DDL writing.  

We don't what this type of change buried in a PR that changes 300 files.  If a PR is going to add @Override everywhere that is OK as long as that is all the PR does.  It is not ok to also include other subtle changes like this.

This needs to be 2 PRs.

**Comment by rbygrave** [comment]
Yes, you need to split this PR or close it.

**Comment by rPraml** [comment]
closed - created new PR that adds only @overrides

**Pull request closed without merging**